docs/buildah-build.1.md

```diff
@@ -96,7 +96,37 @@ The value of `[name]` is matched with the following priority order:
 
 **--cache-from**
 
-Images to utilise as potential cache sources. Buildah does not currently support --cache-from so this is a NOOP.
+Repository to utilise as potential cache source. When specified buildah will try to look for
+cache images in specified repository and will attempt to pull cache images instead of actually
+executing and computing STEPS. Buildah will only attempt to pull cache images if they are considered
+as valid cache hits otherwise not.
+
+Use `--cache-to` in order populate remote repository with valid cache sources/layers.
```

Comment:
```suggestion
Use `--cache-to` in order populate a remote repository with cache content.
```

Reply:
“in order _to_ populate”?

Reply:
Or just “… to populate”, without “in order”